docs/csharp/misc/cs0473.md

```diff
@@ -61,4 +61,4 @@ class T
 
 ## See also
 
-- [Fabulous Adventures in Coding](https://blogs.msdn.microsoft.com/ericlippert/2006/04/06/odious-ambiguous-overloads-part-two/)
+- [Fabulous Adventures in Coding](https://docs.microsoft.com/archive/blogs/ericlippert/odious-ambiguous-overloads-part-two)
```

Comment:
Let's use the chance to change the link text from the blog title to the article title:

```suggestion
- [Odious ambiguous overloads, part two](https://docs.microsoft.com/archive/blogs/ericlippert/odious-ambiguous-overloads-part-two)
```